.travis.yml

```diff
@@ -13,6 +15,7 @@ addons:
 script:
 - export CC=/usr/bin/gcc-6
 - export CXX=/usr/bin/g++-6
+- if rustc -Vv | grep nightly; then cargo check --no-default-features; fi
```

Comment:
I think using `TRAVIS_RUST_VERSION` would be more reliable. Something like this:

```
if [ "$TRAVIS_RUST_VERSION" == "nightly" ]; then cargo build --no-default-features; fi
```

Reply:
Nice! Didn't know about it!